arches_resource_sets/views/resource_set_members_bulk_view.py

```diff
@@ -0,0 +1,33 @@
+from json import JSONDecodeError
+import logging
+
+from django.forms import ValidationError
+from arches.app.utils.betterJSONSerializer import JSONDeserializer
+from arches.app.utils.response import JSONErrorResponse, JSONResponse
+from arches.app.views.api import APIBase
+from arches_resource_sets.models import ResourceSet
+
+logger = logging.getLogger(__name__)
+class ResourceSetMembersBulkView(APIBase):
+    def post(self, request, set_id):
+        try:        
+            request_body = JSONDeserializer().deserialize(request.body)
+            resource_ids = request_body["resource_ids"] if "resource_ids" in request_body else ""
+            operation = request_body["operation"] if "operation" in request_body else "add"
```

Comment:
for consistencies sake let's use the verbose terminology
```suggestion
                request_body["resource_instance_ids"] if "resource_instance_ids" in request_body else ""
```

Reply:
I think what was meant was
```
resource_instance_ids = request_body["resource_instance_ids"] if "resource_instance_ids" in request_body else ""
```